### PR TITLE
Mobile contact section

### PR DESCRIPTION
**Pull request by brighton0102**
in this project, my team and I collaborated on the mobile-contact-section
. we used HTML form elements 
. Added button interaction to the submit button
. we added the input field Name and Email
. we also added the text area 
And used CSS to style the HTML form elements

### CODE REVIEW

**Review by Gambit142 — COMMENTED** [review]
Hi @brighton0102 @richie1988 @zerihunabune 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- Kindly include the Future Features section in your README. This section should contain 1 - 3 potential features that can be implemented in the future to make the project better 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -1,47 +1,70 @@
...
+                </li>
+                <li>
+                    <a href="#"><img src="/img/medium.svg" alt="medium" /></a>
+                </li>
+                <li>
+                    <a href="#"><img src="/img/angelList.svg" alt="angelList" /></a>
+                </li>
+                <li>
+                    <a href="#"><img src="/img/git_hub.svg" alt="gitHub" /></a>
+                </li>
```
Comment (bienvenuushindi) [inline index.html:54]: :orange_book:  [OPTIONAL] None of the images is showing as you can see below :arrow_down:. I think you forgot to push the images to GitHub as they are nonexistent in the image folder. Please make sure you add them

Your Design
image

File: index.html [hunk 2]
```diff
@@ -53,16 +76,18 @@ <h2 class="title-work-section">Tonic</h2>
...
                 <li class="language">JavaScript</li>
             </ul>
-            <a href="#"><button>See project</button></a>
+            <a href="#"><button type="submit">See project</button></a>
         </section>
 
         <section class="cards">
-            <img class="card-works-image" src="/img/Snapshoot_Portfolio_2.svg" alt="card-works">
-            <h2 class="title-work-section">Multi-Post <br> Stories</h2>
+            <img class="card-works-image" src="/img/Snapshoot_Portfolio_2.svg" alt="card-works" />
```
Comment (bienvenuushindi) [inline index.html:83]: :orange_book:  [OPTIONAL] None of the images is showing as you can see below :arrow_down:. I think you forgot to push the images to GitHub as they are nonexistent in the image folder. Please make sure you add them

Your Design
image2

**Review by bienvenuushindi — APPROVED** [review]
Hi team @brighton0102, @richie1988  and @znation2010 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

:heavy_check_mark:  Descriptive Pull Request
:heavy_check_mark:  Validations correctly implemented
:heavy_check_mark: No linter errors

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**